core/commands/pin.go

```diff
@@ -539,7 +539,7 @@ func pinLsKeys(args []string, typeStr string, ctx context.Context, n *core.IpfsN
 	return keys, nil
 }
 
-func pinLsAll(typeStr string, ctx context.Context, n *core.IpfsNode) (map[string]RefKeyObject, error) {
+func pinLsAll(typeStr string, n *core.IpfsNode) (map[string]RefKeyObject, error) {
```

Comment:
Instead of removing this context, I'd use it and pass it to the `EnumerateChildren` call.